moveit_planners/ompl/CHANGELOG.rst

```diff
@@ -2,6 +2,38 @@
 Changelog for package moveit_planners_ompl
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+1.1.0 (2020-09-02)
+------------------
+
+1.0.6 (2020-08-19)
+------------------
+* [maint] Adapt repository for splitted moveit_resources layout (`#2199 <https://github.com/ros-planning/moveit/issues/2199>`_)
+* [maint] Migrate to clang-format-10
+* [maint] Optimize includes (`#2229 <https://github.com/ros-planning/moveit/issues/2229>`_)
+* [fix]   Fix memcpy bug in copyJointToOMPLState in ompl interface (`#2239 <https://github.com/ros-planning/moveit/issues/2239>`_)
+* Contributors: Jeroen, Markus Vieth, Robert Haschke, Michael Görner
+
+1.0.5 (2020-07-08)
+------------------
+* [feature] Added support for hybridize/interpolate flags in ModelBasedPlanningContext via ompl_planning.yaml (`#2171 <https://github.com/ros-planning/moveit/issues/2171>`_, `#2172 <https://github.com/ros-planning/moveit/issues/2172>`_)
+* Contributors: Constantinos, Mark Moll
+
+1.0.4 (2020-05-30)
+------------------
+
+1.0.3 (2020-04-26)
+------------------
+* [maint] Cleanup OMPL dynamic reconfigure config (`#1649 <https://github.com/ros-planning/moveit/issues/1649>`_)
+  * Reduce minimum number of waypoints in solution to 2
+* [maint] Apply clang-tidy fix to entire code base (`#1394 <https://github.com/ros-planning/moveit/issues/1394>`_)
+* [maint] Fix errors: catkin_lint 1.6.7 (`#1987 <https://github.com/ros-planning/moveit/issues/1987>`_)
+* [maint] Windows build: Fix binary artifact install locations. (`#1575 <https://github.com/ros-planning/moveit/issues/1575>`_)
+* [maint] Use CMAKE_CXX_STANDARD to enforce c++14 (`#1607 <https://github.com/ros-planning/moveit/issues/1607>`_)
+* Contributors: Michael Görner, Robert Haschke, Sean Yen, Yu, Yan
+
+1.0.2 (2019-06-28)
+------------------
```

Comment:
@rhaschke here is an example of how it worked how I thought it would.  Looking through some of them it didn't work.  I'll fix that tomorrow morning.  Good catch.

Reply:
That what I had in my mind. Thanks for reworking this!

Reply:
> You said you migrated changelogs from Melodic. If so, wouldn't it be more reasonable to list those changes below corresponding tags 1.0.x and only focus on really new contributions (compared to the last melodic release 1.0.6) below 1.1.0? Now, we have a looong list of changes below 1.1.0.

See my comment.  Looking through the changes I did think some of them were way longer than I expected.  Also, because the history of master and melodic-devel diverged (some old changes are only on master, and some newer ones are on melodic-devel) that makes this a bit more challenging, but I'll get it done.

Reply:
I wrote a script to do this... the short answer is it didn't work completely and I didn't do a good enough job of checking the results... I'll fix this up.